processor/spanprocessor/span.go

```diff
@@ -28,12 +28,15 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector/consumer/consumerdata"
 	"github.com/open-telemetry/opentelemetry-collector/oterr"
 	"github.com/open-telemetry/opentelemetry-collector/processor"
+	"github.com/open-telemetry/opentelemetry-collector/processor/common"
 )
 
 type spanProcessor struct {
 	nextConsumer     consumer.TraceConsumer
 	config           Config
 	toAttributeRules []toAttributeRule
+	include          common.MatchingProperties
+	exclude          common.MatchingProperties
```

Comment:
Nit: extract the pair as one entity and reuse in both processors. This will ensure there are no discrepancies in behavior.